flink-end-to-end-tests/test-scripts/test_quickstarts.sh

```diff
@@ -51,12 +59,12 @@ sed -i -e ''"$(($position + 1))"'i\
 <version>${flink.version}</version>\
```

Comment:
It would be _awesome_ if you could replace this hard-coded dependency with the one defined in the example project. But I wouldn't block the PR on it.

Reply:
I did some code change to get the ES dependency from flink-quickstart-test/pom.xml and removed the hardcoded values